packages/models-library/src/models_library/service_settings_labels.py

```diff
@@ -149,6 +161,10 @@ class Config(_BaseConfig):
 
 SimcoreServiceSettingsLabel = ListModel[SimcoreServiceSettingLabelEntry]
 
+#
+# "simcore.service.paths-mapping" label
+#
```

Comment:
why don't you put these as docstring instead of a flying comment? it is already separating itself form the class...